bin/ephembot

```diff
@@ -1,8 +1,25 @@
 #!/usr/bin/env node
 require('babel/register')
-var app = require('../index.js')
+
+var IoC = require('electrolyte')
+var path = require('path')
+
+/**
+ * Parse the CLI
+ */
 var argv = require('minimist')(process.argv.slice(2))
 
+IoC.loader(IoC.node(path.resolve(__dirname, '..')))
```

Comment:
This tells the Inversion of Control container where to look for dependency annotations.